soul-client/soul-client-http/soul-client-springcloud/src/test/java/org/dromara/soul/client/springcloud/init/SpringCloudClientBeanPostProcessorTestConfig.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dromara.soul.client.springcloud.init;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+import org.springframework.beans.factory.support.BeanDefinitionBuilder;
+import org.springframework.beans.factory.support.BeanDefinitionRegistry;
+import org.springframework.beans.factory.support.BeanDefinitionRegistryPostProcessor;
+import org.springframework.context.annotation.Bean;
+
+/**
+ * Test Case for SpringCloudClientBeanPostProcessor.
+ *
+ * @author DaveModl (davemo-coderpersonal@hotmail.com)
+ */
+public final class SpringCloudClientBeanPostProcessorTestConfig {
```

Comment:
Why we need this？

Reply:
init bean SpringCloudClientBeanPostProcessor

Reply:
In this case. We only need to test `SpringCloudClientBeanPostProcessor`. There is no need to verify Spring's bean initialization. You can just new/mock  one instance.

Reply:
In this case. We only need to test SpringCloudClientBeanPostProcessor. There is no need to verify Spring's bean initialization. You can just new/mock one instance.